golang/cosmos/x/swingset/keeper/querier.go

```diff
@@ -68,7 +68,7 @@ func queryStorage(ctx sdk.Context, path string, req abci.RequestQuery, keeper Ke
 		return []byte{}, sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, "could not get storage")
 	}
 
-	bz, err2 := codec.MarshalJSONIndent(legacyQuerierCdc, types.Storage{value})
+	bz, err2 := codec.MarshalJSONIndent(legacyQuerierCdc, types.Storage{Value: value})
```

Comment:
Fix the construction on line 107 too and you will have fixed all the `go vet` problems - but run it again to be sure.

Reply:
Addressed in #3696